Jenkinsfile

```diff
@@ -12,8 +12,8 @@ pipeline {
     KINTO_DOTENV = credentials('KINTO_DOTENV')
   }
   triggers {
-    pollSCM('H/5 * * * *')
-    cron('H H * * *')
+    pollSCM(env.BRANCH_NAME == 'master' ? 'H/5 * * * *' : '')
```

Comment:
I'd vote for removing this, as multibranch pipeline will detect changes when it scans.

Reply:
On second thoughts, mutlibranch is only in use on preprod, so maybe let's keep it.